inspirehep/bat/pages/holding_panel_literature_list.py

```diff
@@ -49,8 +49,28 @@ def click_first_record():
             (By.XPATH, '(//div[@class="ng-scope"])[2]')))
 
 
-def load_submission_record(input_data):
-    def _load_submission_record():
+def load_submitted_record():
+    def _force_load_record():
+        try:
+            # click the abstract link
```

Comment:
What is the click on the abstract used for? You use the content inside later for assertions?